### PR TITLE
Replace asterisk in table with paragraph

### PR DESCRIPTION
**Pull request by anbj**
Ref. #99. Some datasets may be missing this information,

### CODE REVIEW

**Comment by anbj** [comment]
Argh, I forgot to change branch after my previous fix. Problem?

File: docs/earth-age.rst [hunk 1]
```diff
@@ -25,8 +25,6 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_age:
 
-.. table:: EarthByte Earth Seafloor Crustal Age. An asterisk denotes tiled datasets.
```
Comment (seisman) [inline docs/earth-age.rst:28]: Why remove the table title? 
```
.. table:: EarthByte Earth Seafloor Crustal Age.
```
should be kept.

**Comment by anbj** [comment]
Okay, thought it wasn't necessary.

**Comment by seisman** [comment]
I think you're right. These table titles are not necessary. I'm OK with either keeping or removing them.

**Comment by anbj** [comment]
Ok, I would prefer to remove them then. I think I added those titles/captions when I introduced the asterisk.

**Comment by joa-quim** [comment]
Yes, those titles are redundant and can be removed.

But one thing that is not good is on the How to find the datasets. Some GMT versions ago that was changed and now is challenging to find that page.

On the main page https://docs.generic-mapping-tools.org/latest/
we have to click
https://docs.generic-mapping-tools.org/latest/datasets.html
then
https://docs.generic-mapping-tools.org/latest/datasets/remote-data.html#currently-available-remote-data-sets
and then scroll and find the kink
https://www.generic-mapping-tools.org/remote-datasets/
that finally land on the datasets

**Comment by seisman** [comment]
> But one thing that is not good is on the How to find the datasets.

Add a link in the "Quick Links"?

**Comment by joa-quim** [comment]
Not good either (too crowded).
Need more thinking but I think the entry point
https://docs.generic-mapping-tools.org/latest/
should point directly to
https://www.generic-mapping-tools.org/remote-datasets/
then the documentation
https://docs.generic-mapping-tools.org/latest/datasets.html
should be accessible (at the bottom) directly either from 
https://www.generic-mapping-tools.org/remote-datasets/
or via some other path.

**Comment by seisman** [comment]
> Not good either (too crowded). Need more thinking but I think the entry point https://docs.generic-mapping-tools.org/latest/ should point directly to https://www.generic-mapping-tools.org/remote-datasets/ then the documentation https://docs.generic-mapping-tools.org/latest/datasets.html should be accessible (at the bottom) directly either from https://www.generic-mapping-tools.org/remote-datasets/ or via some other path.

Sounds a good idea. Anyway, it should be addressed in a separate PR.

**Review by seisman — APPROVED** [review]

**Pull request merged**